st2tests/st2tests/fixtures/packs/dummy_pack_16/fixture.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from st2tests import fixturesloader
+
+PACK_NAME, PACK_PATH = fixturesloader.get_fixture_name_and_path(__file__)
```

Comment:
This one did not need special vars. Instead, it needed special handling in one of the tests that counted the number of files in the pack.